scripts/ci/libraries/_kind.sh

```diff
@@ -195,6 +197,16 @@ function kind::perform_kind_cluster_operation() {
             echo "Entering an interactive shell for kubernetes testing"
             echo
             "${AIRFLOW_SOURCES}/scripts/ci/kubernetes/ci_run_kubernetes_tests.sh" "-i"
+        elif [[ ${OPERATION} == "k9s" ]]; then
+            echo
+            echo "Starting k9s CLI"
+            echo
+            export TERM=xterm-256color
+            export EDITOR=vim
+            export K9S_EDITOR=vim
+            docker run --rm -it --network host -e COLUMNS="$(tput cols)" -e LINES="$(tput lines)" \
+                -e EDITOR -e K95_EDITOR \
```

Comment:
```suggestion
                -e EDITOR -e K9S_EDITOR \
```

Reply:
:D

Reply:
Just for a second, 95 reminded me of Windows 95 :D 

![https://media.giphy.com/media/lcYkeZa1gBCzS/giphy.gif](https://media.giphy.com/media/lcYkeZa1gBCzS/giphy.gif)

Reply:
You know one of my first IT jobs was to help with Win95 translation to Polish :)

Reply:
Wow, that must be something. I still like Win 95 and Win 98 compared to Vista 😄

Reply:
I do not like either :)